packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js

```diff
@@ -0,0 +1,343 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Sets the correct path for the medium variation
+ *
+ * @type {string}
+ * @private
+ */
+const _pathMedium = '/iframe.html?id=components-feature-card--default';
+
+/**
+ * Sets the correct path for the large variation
+ *
+ * @type {string}
+ * @private
+ */
+const _pathLarge = '/iframe.html?id=components-feature-card--large';
+
+/* eslint-disable cypress/no-unnecessary-waiting */
+describe('Feature Card | medium', () => {
+  beforeEach(() => {
+    cy.visit(`/${_pathMedium}`);
+    cy.viewport(1280, 780);
+  });
+
+  it('should check for link', () => {
+    cy.get('.bx--feature-card a.bx--link').then($link => {
+      const url = $link.prop('href');
+      expect(url).not.to.be.empty;
+    });
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
+    // cy.percySnapshot('FeatureCard | has link', {
+    //   widths: [1280],
+    // });
+  });
+
+  it("should check that the footer's pseudo class takes up entire card to be clickable", () => {
+    cy.get('.bx--feature-card a.bx--link').then($els => {
+      const win = $els[0].ownerDocument.defaultView;
+      const after = win.getComputedStyle($els[0], 'after');
+      const positionValue = after.getPropertyValue('position');
+      const insetValue = after.getPropertyValue('inset');
+
+      expect(positionValue).to.eq('absolute');
+      expect(insetValue).to.eq('0px');
+    });
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
```

Comment:
Looks good!! only small thing is changing the comments for react if there any percy snapshots we want there